build.gradle

```diff
@@ -40,7 +44,6 @@ configurations.all {
 
 dependencies {
     /// dependencies for app building
-    compile project(':touchImageView')
```

Comment:
Btw, I was annoyed by no maintenance of this lib. But after going my own way and having a vital fork, Mike Ortiz made me to a maintainer and as you can see, there is now some life there https://github.com/MikeOrtiz/TouchImageView/graphs/contributors,
but anyway, if you are happy with `com.github.chrisbanes:PhotoView` it's perfect !
@michaelstingl @davigonz

Reply:
We decided to use PhotoView because, as you can see in the files changed tab, we have pretty less code.

Reply:
It's a library, and in library it's one **one** relevant file.